solutions/instances/variables.tf

```diff
@@ -289,7 +296,7 @@ variable "existing_at_cos_target_bucket_endpoint" {
 
 variable "skip_cos_kms_auth_policy" {
   type        = bool
-  description = "To skip creating an IAM authorization policy that allows the created Cloud Object Storage instance to read the encryption key from the key management service (KMS) instance, set this variable to `true`. Before you can create an encrypted Cloud Object Storage bucket, an authorization policy must exist."
+  description = "Whether to create an IAM authorization policy that permits the Object Storage instance to read the encryption key from the KMS instance. An authorization policy must exist before an encrypted bucket can be created. Set to `true` to avoid creating the policy. If set to `false`, specify a value for `ibmcloud_kms_api_key`."
```

Comment:
If set to `false`, specify a value for `ibmcloud_kms_api_key`.

^^ thats not right? `ibmcloud_kms_api_key` is used when KMS in a different account - it has no impact on the functionality that the `skip_cos_kms_auth_policy` variable has

Reply:
Another one that I had suggested. Looks like it got resolved before you had a chance to review, Conall. 

> This is closer to other `skip_` descriptions. Make sure it's correct. @ocofaigh, I don't think we need the cross-account information because we have it elsewhere where you do the work. But let me know if that's wrong.